include/ultra64/exception.h

```diff
@@ -0,0 +1,45 @@
+#ifndef ULTRA64_EXCEPTION_H
+#define ULTRA64_EXCEPTION_H
+
+#include "types.h"
+
+// Interrupt masks
+#define OS_IM_NONE      0x00000001
+#define OS_IM_SW1       0x00000501
+#define OS_IM_SW2       0x00000601
+#define OS_IM_CART      0x00000c01
+#define OS_IM_PRENMI    0x00001401
+#define OS_IM_RDBWRITE  0x00002401
+#define OS_IM_RDBREAD   0x00004401
+#define OS_IM_COUNTER   0x00008401
+#define OS_IM_CPU       0x0000ff01
+#define OS_IM_SP        0x00010401
+#define OS_IM_SI        0x00020401
+#define OS_IM_AI        0x00040401
+#define OS_IM_VI        0x00080401
+#define OS_IM_PI        0x00100401
+#define OS_IM_DP        0x00200401
+#define OS_IM_ALL       0x003fff01
```

Comment:
we've been using uppercase for hex
also with `OS_IM_CPU`, `RCP_IMASK`
and in `r4300.h`